apps/web/components/booking/BookingListItem.tsx

```diff
@@ -393,7 +400,9 @@ function BookingListItem(booking: BookingItemProps) {
         <td className="py-4 pl-4 text-right text-sm font-medium ltr:pr-4 rtl:pl-4 sm:pl-0">
           {isUpcoming && !isCancelled ? (
             <>
-              {isPending && user?.id === booking.user?.id && <TableActions actions={pendingActions} />}
+              {isPending && booking.user.some((bookingUser) => bookingUser.id === user?.id) && (
+                <TableActions actions={pendingActions} />
+              )}
```

Comment:
Allow all members of a team to accept / reject bookings

Reply:
Is this by design?